tools/test-setup.sh

```diff
@@ -379,10 +379,11 @@ if [[ "${DOCKER_VERSION}" != 'null' ]] && [[ "${SKIP_DOCKER:-}" != '1' ]]; then
     EE_ANSIBLE_VERSION=$(get_version \
         docker run --rm "${IMAGE}" ansible --version)
     EE_ANSIBLE_LINT_VERSION=$(get_version \
-        docker run --rm "${IMAGE}" ansible-lint --nocolor --version)
-    # Test podman ability to mount current folder with write access, default mount options
-    docker run --rm -v "$PWD:$PWD" ghcr.io/ansible/creator-ee:latest \
-        bash -c "[ -w $PWD ] && echo 'Mounts working' || { echo 'Mounts not working. You might need to either disable or make selinux permissive.'; exit 1; }"
+        docker run "${IMAGE}" ansible-lint --nocolor --version)
+    log notice "ansible: ${EE_ANSIBLE_VERSION}, ansible-lint: ${EE_ANSIBLE_LINT_VERSION}"
+    # Test docker ability to mount current folder with write access, default mount options
+    docker run -v "$PWD:$PWD" ghcr.io/ansible/community-ansible-dev-tools:latest \
```

Comment:
although this could work, I think long term we'd want a writable mount.  The image should default to the "podman" user but that could be checked.  It might help to ls -l the dir here to see the perms

Reply:
interesting, the file for docker are owned by 1001, wehich tells be the runner user is not 1000.  Maybe try a user-podman for docker.

(which is odd thinking about it now https://github.com/ansible/ansible-dev-tools/issues/334)

Reply:
keeping this as is per discussion on weekly call